build/tests.mk

```diff
@@ -49,6 +49,39 @@ crypto-tests = \
 	${1} test/coconut_test.lua && \
 	${1} test/crypto_abc_zeta.lua
 
+cortex-m-crypto-tests = \
+	${1}test/octet.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/octet_conversion.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/hash.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/ecdh.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/dh_session.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/nist/aes_gcm.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/nist/aes_cbc.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/nist/aes_ctr.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/ecp_generic.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/elgamal.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/bls_pairing.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/coconut_test.lua && cat ./outlog &&\
+	rm ./outlog && ${1}test/coconut_abc_zeta.lua && cat ./outlog
+
+cortex-m-zencode-integration = \
+	cd test/zencode_numbers && ./run.sh ${1}; cd -; \
+	cd test/zencode_array && ./run.sh ${1}; cd -; \
+	cd test/zencode_hash && ./run.sh ${1}; cd -; \
+	cd test/zencode_ecdh && ./run.sh ${1}; cd -; \
+	cd test/zencode_credential && ./run.sh ${1}; cd -; \
+	cd test/zencode_petition && ./run.sh ${1}; cd -;
+	# not pass yet
```

Comment:
Since there are remaining test items which would expectedly fail, you should mention in git commit message(s).

Reply:
with explanation explicitly.

Reply:
Some test files don't pass yet. It will be modified in another PR to be passed.